src/static/css/content.less

```diff
@@ -765,3 +765,32 @@ ul {
 }
 // Initiate loop
 .columnX ( @grid_total-columns );
+
+
+/* Page components/Tables page
+   ========================================================================== */
+
+// Display all example tables at 100% width
+table {
+    width: 100%;
+}
+
+// Display pagination examples in half columns as their mobile view
```

Comment:
I still need to try the iframe suggestion @niqjohnson had. Maybe I can find time on Friday.